src/coreclr/vm/frames.cpp

```diff
@@ -464,6 +464,21 @@ void Frame::PopIfChained()
 }
 #endif // TARGET_UNIX && !DACCESS_COMPILE
 
+#if !defined(TARGET_X86) || defined(TARGET_UNIX)
+void Frame::UpdateFloatingPointRegisters(const PREGDISPLAY pRD, TADDR targetSP)
+{
+    _ASSERTE(!ExecutionManager::IsManagedCode(::GetIP(pRD->pCurrentContext)));
+    while (!ExecutionManager::IsManagedCode(::GetIP(pRD->pCurrentContext)))
```

Comment:
Do all places where this is used guarantee that the managed code is the right stopping point?

Reply:
For example, consider the case of managed code called from CallDescWorker.